### DISCUSSION TITLE
Changing colors in Timeline instead of 50 shades of green :D

### DISCUSSION THREAD

**Discussion by gavishgulati** [certinia/debug-log-analyzer#27] [Ideas] [answered]
Hi Team,

First and foremost, thanks a lot for building this. This tool has saved my eyes, I was confident that I would go blind by the age of 35 after looking at debug logs as is everyday!

A different color scheme for the timeline would really help in identifying each code unit. Color scheme that salesforce has used for their Timeline viewer isn't too bad so maybe that can be replicated.

Thanks again for building this extension!

**Comment by nawforce** [comment]
HI Gavish, 

Glad you are enjoying the tool, I also spend way too much time with debug logs so am very familiar with that pain.

Thanks for the suggestion of using the Salesforce timeline viewer colours, I had not thought about comparing.  This colour question has come up a few times, I think we have concluded that we need to at least support a few different colours schemes and perhaps also allow people to customise. 

We have done some work to make the code easier for other people to work on but I think there is a bit more to do on that. Once we have that completed I think we will look to add this as a high priority item and get an update released.

**Reply by lcottercertinia** [reply]
Hi @gavishgulati 
There is a setting for this now, you can chose whatever colors you like, just find the hex color code

preferences -> settings -> extensions -> Apex Log Analyzer -> Timeline: Colors
![image](https://user-images.githubusercontent.com/81575432/161027805-30a6b833-c2bf-4479-91cc-60661d897d60.png)

Hope this helps

**Comment by lcottercertinia** [comment, marked as answer]
There is a setting for this now, you can chose whatever colors you like, just find the hex color code

preferences -> settings -> extensions -> Apex Log Analyzer -> Timeline: Colors
![image](https://user-images.githubusercontent.com/81575432/161027805-30a6b833-c2bf-4479-91cc-60661d897d60.png)

Hope this helps

**Reply by lcottercertinia** [reply]
In addition we are also tweaking the default colours (#230  ) to be more easily distinguishable
